src/components/DataTable/utils/renderHelpers.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {Chip} from '~/index';
+
+export type LocaleOptions = {
+    locale?: string | string[];
+    numberOptions?: Intl.NumberFormatOptions;
+    dateOptions?: Intl.DateTimeFormatOptions;
+};
+
+export const renderString = () =>
+    (value: string | { value: string }): React.ReactNode =>
+        typeof value === 'string' ? value : value.value;
+
+export const renderNumber = (options?: LocaleOptions) =>
```

Comment:
All render functions should take a `value` parameter in order to be used in many contexts instead of being only usable in `columnHelpers` functions.

```
renderNumber(value, locale, localeOptions)
```